src/locales/en.json

```diff
@@ -170,5 +170,6 @@
 	"CSV_UPLOADED_SUCCESSFULLY": "Sample Session Creation CSV Uploaded Successfully",
 	"CSV_UPDATE_FAILED": "Sample Session Creation CSV Upload Failed",
 	"SESSION_CSV_UPLOADED_FAILED": "Bulk Session Creation CSV Upload Failed",
-	"CSV_ROW_LIMIT_EXCEEDED": "CSV row limit exceeded. Please reduce the number of rows in your CSV file."
+	"CSV_ROW_LIMIT_EXCEEDED": "You can only upload 20 sessions at a time",
```

Comment:
The limit is configurable right? So how can we hardcode this in the response?

Reply:
will change it